docs-gen/content/rule_set/data_entry/_index.md

```diff
@@ -13,9 +13,9 @@ a publisher (or producer) that continuously updates the signal value while an
 attribute has a set value, defined in the specification, that never changes.
 As summary, besides [```branch```](/rule_set/branches) type can be:
 
-* **```attribute```**, which describes static read-only value.
-* **```sensor```**, which describes non-static read-only signal.
-* **```actuator```**, which describes a non-static read-write signal.
+* **```attribute```**: attributes are not expected to change once they're set (e.g. vehicle identification number)
+* **```sensor```**: sensor values describe the current state of the vehicle and change over time, as the state of the vehicle changes (e.g. odometer).
```

Comment:
Also short and good, only I'm concerned with having it general enough.
There's some big movements going on as you know in CVII and so on, so in terms of a data model I'm not even sure that the VSS model is only describing "the vehicle" in the future.  But we can decide on that after clarifying the overall "from which viewpoint" discussion.